charts/aws-pca-issuer/templates/hpa.yaml

```diff
@@ -14,20 +14,23 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: {{ include "aws-privateca-issuer.fullname" . }}
-    namespace: {{ .Release.Namespace }}
   minReplicas: {{ .Values.autoscaling.minReplicas }}
   maxReplicas: {{ .Values.autoscaling.maxReplicas }}
   metrics:
     {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
     - type: Resource
       resource:
         name: cpu
-        targetAverageUtilization: {{ .Values.autoscaling.targetCPUUtilizationPercentage }}
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetCPUUtilizationPercentage }}
```

Comment:
This is backwards incompatible with `autoscaling/v2beta1` which is used in K8s <1.22. We will need to keep `targetAverageUtilization` for clusters running `autoscaling/v2beta1`, and use this for clusters running `autoscaling/v2`.

Reply:
```
helm install issuer ./charts/aws-pca-issuer -n aws-privateca-issuer \
--set serviceAccount.create=false --set serviceAccount.name=aws-privateca-issuer-sa \
--set image.repository="localhost:5000/aws-privateca-issuer" --set image.tag=latest --set image.pullPolicy=Always \
--set autoscaling.enabled=true
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(HorizontalPodAutoscaler.spec.metrics[0].resource): unknown field "target" in io.k8s.api.autoscaling.v2beta1.ResourceMetricSource
make: *** [install-local] Error 1
```